src/main/scala/is/hail/rvd/UnpartitionedRVD.scala

```diff
@@ -10,13 +10,25 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.storage.StorageLevel
 
 object UnpartitionedRVD {
-  def empty(sc: SparkContext, rowType: TStruct): UnpartitionedRVD =
-    new UnpartitionedRVD(rowType, ContextRDD.empty[RVDContext, RegionValue](sc))
+  def empty(sc: SparkContext, rowType: TStruct): RVD =
+    UnpartitionedRVD(rowType, ContextRDD.empty[RVDContext, RegionValue](sc))
+
+  def apply(rowType: TStruct, crdd: ContextRDD[RVDContext, RegionValue]): RVD = {
+    new UnpartitionedRVD(rowType, crdd)
+  }
 }
 
 class UnpartitionedRVD(val rowType: TStruct, val crdd: ContextRDD[RVDContext, RegionValue]) extends RVD {
```

Comment:
UnpartitionedRVD is on its way out. I added the apply method when I was considering how to migrate away from UnpartitionedRVD. The thought was that it would be useful to have a way to toggle the UnpartitionedRVD factory method to actually return an OrderedRVD. So yeah, I should probably make the constructor private.